client/src/pages/CustomerProfile.js

```diff
@@ -157,9 +187,28 @@ const CustomerProfile = () => {
           </Grid>
         </Grid>
         <div style={{ height: "45%", width: "100%" }}>
-          <Map lat={userData.location.lat} lng={userData.location.lng}></Map>
+          <Map
+            lat={
+              userData.location && userData.location.coordinates
+                ? userData.location.coordinates[1]
```

Comment:
I don't think the ternary operator works both with '&&' and '?'. I think It only accepts either '&&' or an '?' in one condition

Reply:
it works @Vaibhav-Kambli  it checks for truthy of the whole expression before ? and then does the rest